src/integration-tests/introspector/introspectTest.sh

```diff
@@ -99,6 +99,21 @@ export ALLOW_DYNAMIC_CLUSTER_IN_FMW=${ALLOW_DYNAMIC_CLUSTER_IN_FMW:-false}
 # whether this test run is expecting a domain validation error
 export EXPECT_INVALID_DOMAIN=${EXPECT_INVALID_DOMAIN:-false}
 
+DOMAIN_SOURCE_TYPE=${DOMAIN_SOURCE_TYPE:-Image}
+export DOMAIN_SOURCE_TYPE=${DOMAIN_SOURCE_TYPE}
+WDT_DOMAIN_TYPE=${WDT_DOMAIN_TYPE:-WLS}
+export WDT_DOMAIN_TYPE=${WDT_DOMAIN_TYPE}
```

Comment:
WDT_DOMAIN_TYPE only needed for FromModel path.

Reply:
I changed the yaml template file to have the environment variable

Reply:
Note that the yamlt template files can handle default macros - you can modify them so that `${WDT_DOMAIN_TYPE}` becomes `${WDT_DOMAIN_TYPE:-WLS}`.

Reply:
Once you do that, you don't need to set WDT_DOMAIN_TYPE globally to something else unless going through the MII path...

Reply:
done